src/main/java/hong/Hong.java

```diff
@@ -0,0 +1,69 @@
+package hong;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import parsers.Parser;
+import tasks.Task;
+import tasks.TaskList;
+import ui.Ui;
+
+
```

Comment:
Consider adding a JavaDoc here as it is an important class for this application.